### PR TITLE
[docs] v0.11 What's new

### PR DESCRIPTION
**Pull request by inikulin**
\cc @DevExpress/testcafe-docs @lexkazakov

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,226 @@
...
+
+*Example test code:*
+
+```js
+test('Button click', async t => {
+    const btn = Selector('#btn');
+
+    await t
+        .click(btn)
+        // NOTE: Regular assertion will fail immediately, but TestCafe retries to run DOM state
```
Comment (helen-dikareva) [inline CHANGELOG.md]: we never use `//NOTE:` prefix in documentation

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,225 @@
...
+
+TestCafe now ships with [numerous built-in BDD-style assertions](http://devexpress.github.io/testcafe/documentation/test-api/assertions/assertion-api.html).
+If the TestCafe assertion receives a [Selector's property](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html#members-common-across-all-nodes) as an actual value, TestCafe uses the [smart assertion query mechanism](http://devexpress.github.io/testcafe/documentation/test-api/assertions/index.html#smart-assertion-query-mechanism):
+if an assertion did not passed, the test does not fail immediately. The assertion retries to pass multiple times and each time it re-requests the actual shorthand value. The test fails if the assertion could not complete successfully within a timeout.
+This approach allows you to create stable tests that lack random errors and decrease the amount of time required to run all your tests due to the lack of extra waitings.
+
+*Example page markup:*
+
+```html
+<div id="btn"></div>
```
Comment (inikulin) [inline CHANGELOG.md]: It shouldn't have text, otherwise what test will check then?

**Comment by inikulin** [comment]
FPR

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,226 @@
...
+* Cookies set via XHR response now available from client code ([testcafe-hammerhead#905](https://github.com/DevExpress/testcafe-hammerhead/issues/905)).
+* Fixed client rendering problems caused by incorrect DOM element determination ([testcafe-hammerhead#953](https://github.com/DevExpress/testcafe-hammerhead/issues/953)).
+
 ## v0.10.0 (2016-11-8)
 
 ### Enhancements
 
-* **Snapshot API shorthands. ([#771](https://github.com/DevExpress/testcafe/issues/771))**
-  
+#### :gear: Snapshot API shorthands. ([#771](https://github.com/DevExpress/testcafe/issues/771))
```
Comment (helen-dikareva) [inline CHANGELOG.md]: Aa far as I know we decided not to use `Snapshot API shorthands` term

File: CHANGELOG.md [hunk 4]
```diff
@@ -53,13 +268,15 @@
   }
   ```
 
-  See [Snapshot API Shorthands](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#snapshot-api-shorthands).  
-* **Improved automatic wait mechanism. ([#245](https://github.com/DevExpress/testcafe/issues/245))**
+  See [Snapshot API Shorthands](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#snapshot-api-shorthands).
```
Comment (helen-dikareva) [inline CHANGELOG.md]: The same as above and there is no `#snapshot-api-shorthands` in `selectors.md`

File: docs/articles/blog/2016-12-8-testcafe-v0-11-0-released.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+* [selectorOptions.dependencies](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selector-options.html#optionsdependencies) - use [filtering](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#filter-multiple-dom-nodes) and [hierarchical](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#find-elements-by-dom-hierarchy) methods to build combined selectors instead.
+
+### ⚙ Built-in assertions. ([#998](https://github.com/DevExpress/testcafe/issues/998))
+
+TestCafe now ships with [numerous built-in BDD-style assertions](http://devexpress.github.io/testcafe/documentation/test-api/assertions/assertion-api.html).
+If the TestCafe assertion receives a [Selector's property](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html#members-common-across-all-nodes) as an actual value, TestCafe uses the [smart assertion query mechanism](http://devexpress.github.io/testcafe/documentation/test-api/assertions/index.html#smart-assertion-query-mechanism):
```
Comment (helen-dikareva) [inline docs/articles/blog/2016-12-8-testcafe-v0-11-0-released.md]: We gave [Selector DOM node state property](../selecting-page-elements/selectors.md#define-assertion-actual-value) link in this sentence in assertion documentation, let's chose one variant for this

**Comment by inikulin** [comment]
FPR \cc @lexkazakov @DevExpress/testcafe-docs I've also changed PageObject example for selectors.

**Comment by lexkazakov** [comment]
looks OK for me

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Pull request merged**